freetype/__init__.py

```diff
@@ -961,12 +962,12 @@ class Face( object ):
     FreeType root face class structure. A face object models a typeface in a
     font file.
     '''
-    def __init__( self, filename, index = 0 ):
+    def __init__( self, path_or_stream, index = 0 ):
```

Comment:
path_or_file would also be ok.

Reply:
Had that before, but technically I expect a stream here...